migrate-abs-to-spe.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Sample App To Migrate from Azure Blob Storage container to SharePoint Embedded container 
+description: Tutorial in how to migrate from Azure Blob Storage container to SharePoint Embedded container
+ms.date: 07/31/2024
+ms.localizationpriority: high
+---
+
+# Introduction
+
+## Purpose
+
+This tutorial will guide you through the process of migrating content from Azure Blob Storage (ABS) to SharePoint Embedded (SPE). This is particularly useful for smaller customers who need to move their content efficiently.
+
+## Prerequisites
+
+1. .NET Core SDK
+1. Dotnet environment to run sample app
+    - It can be run Windows, Linux and MacOs
+1. SharePoint Embedded container
+    - For more information how to set up [SPE container](http://aka.ms/start-spe)
+1. Azure Blob Storage container
+    - For more information how to set up [ABS container](https://learn.microsoft.com/azure/storage/blobs/storage-blobs-introduction)
+
+# Authenication
+
+## Azure Blob Storage
+
+1. Credentials - Container level Shared Access Signature (SAS) URL.
```

Comment:
This starts to feel like a bullet point list rather than a tutorial. Make it a story, add link on how to get the SAS URL. 

Use lists when appropriate. This shouldn’t be a list.